addons/website_slides/controllers/main.py

```diff
@@ -626,6 +591,51 @@ def _slide_channel_prepare_values(self, **kw):
             'allow_comment': bool(kw.get('allow_comment')),
         }
 
+    def slide_channel_prepare_review_values(self, channel):
```

Comment:
Can you make it a private method?
Even though I'm not sure it's needed in controllers.

Reply:
It's not necessary for the controllers, but indeed it's better to make it private :)